packages/foreman/rubygem-sequel/rubygem-sequel.spec

```diff
@@ -66,6 +69,10 @@ find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 %doc %{gem_instdir}/doc
 
 %changelog
+* Thu Mar 09 2023 Bernhard Suttner <suttner@atix.de> 5.66.0-2
+- bigdecimal is a requirement of sequel
+
+- Update to 5.66.0
```

Comment:
I think this may be a copy paste error:
```suggestion
```

I can recommend `rpmdev-bumpspec -c '- bigdecimal is a requirement of sequel' rubygem-sequel.spec` if you're doing it manually now.